jrnl/Journal.py

```diff
@@ -234,8 +234,15 @@ def filter(
 
         self.entries = result
 
+    def delete_entries(self, entries_to_delete):
+        """Deletes specific entries from a journal."""
+        self.entries = [
+            entry for entry in self.entries if entry not in entries_to_delete
+        ]
```

Comment:
Hi! What about deleting only the affected entries without iterate the entire list?

```suggestion
        for entry in entries_to_delete:
            self.entries.remove(entry)
```
This is much faster.